### DISCUSSION TITLE
Some doubts about image resizing during training

### DISCUSSION THREAD

**Discussion by lamiae00r** [buliasz/tesstrain-windows-gui#38] [Q&A] [answered]
hi, first of all thank you for making this a public repository, it is very helpful.

i wanted to ask if there is some sort of resizing of the image during the training process. and also, when using the Ground Truth Preview, is the image resized before being evaluated with the new model? i'm asking because when i evaluate my pictures from my cmd, using the newly trained model, i get outputs like "Warning: Invalid resolution 0 dpi. Using 70 instead. Estimating resolution as 395. Empty page!!". however, when i resize my image to bigger dimensions, i sometimes obtain the same output as when using Ground Truth Preview.

thank you again, regards.

**Comment by Blu3train** [comment]
"Warning: Invalid resolution 0 dpi. Using 70 instead..."
It's the same problem discussed in issue #34. Try reading. I guess the problem was that he was using an older version of Tesseract, but I'm going from memory

**Reply by lamiae00r** [reply]
Thank you for your response @Blu3train

**Comment by buliasz** [comment, marked as answer]
Hi,

Thank you for asking and thank you @Blu3train for helping with answer. In fact the similar issue was because old version of Tesseract was used and I suggest installing the one from [Tesseract at UB Mannheim](https://github.com/UB-Mannheim/tesseract/wiki) repository.

For the resizing question there are two separate things here. The training process is just execution of the original training executables installed with your Tesseract binaries (usually). I'm not sure how it exactly works there with pictures and it may probably change from version to version. This would be a question for the main tesstrain/Tesseract repository.

The other thing is the Ground Truth Preview feature introduced in the GUI. When it shows pictures it fits it to width of the GUI, but it is not modifying the original in any way.

**Reply by lamiae00r** [reply]
Hi, 

Thank you for your reply! It is helpful to know that, I will update my Tesseract and see how the results change! 

Also, if there's anyone encountering the same problem, I realized that I was not indicating --psm 13 (which is the page segmentation mode I had trained with) when executing Tesseract later on, and the results and accuracy improved very much after adding this.
